src/api/BackendConnector.ts

```diff
@@ -1,25 +1,41 @@
-import { Session } from "inspector";
+import { SessionSend, SessionGet } from "./Session";
+import { ParkerStatistic } from "./ParkerSatistic";
 
 class BackendConnector {
-    static createSession(userID: number, kennzeichen: string) {
-        const currentDate = new Date();
-        //TODO
+    static apiUrl: string = `${window.location.protocol}//${window.location.hostname}:3000`
 
-        console.log("backendconnector", userID, kennzeichen)
+    static createSession(userID: number | null, kennzeichen: string) {
+        let newSession: SessionSend = {
+            session: {
+                license_plate: kennzeichen,
+                permanent_parker_id: userID
+            }
+        }
+
+        fetch(`${this.apiUrl}/api/sessions`, 
+            {
+                method: "POST",
+                body: JSON.stringify(newSession),
+                mode: "cors"
```

Comment:
Bei mir hat es weiterhin nicht funktioniert…

```suggestion
                method: "POST",
                headers: {
                  'Content-Type': 'application/json'
                },
                mode: "cors",
                body: JSON.stringify(newSession)
```

Hiermit aber schon.
Magst du die Lösung bitte auch an den anderen stellen implementieren?